score-api/src/main/resources/io/cloudslang/schema/score.xsd

```diff
@@ -21,6 +21,7 @@
             <xsd:attribute type="xsd:boolean" name="externalDatabase" />
             <xsd:attribute type="xsd:boolean" name="ignoreEngineJobs" />
             <xsd:attribute type="xsd:boolean" name="registerPauseResumeService"/>
+            <xsd:attribute type="xsd:boolean" name="registerWorkerNodeService"/>
```

Comment:
We are extending the WorkerNodesService in OO and like in PauseResume case want to be able to load extended implementation

From: Eliya Sadan [mailto:notifications@github.com]
Sent: Wednesday, October 28, 2015 8:04 AM
To: CloudSlang/score
Cc: Beck, Natasha
Subject: Re: [score] Worker version support (#64)


In score-api/src/main/resources/io/cloudslang/schema/score.xsd<https://github.com/CloudSlang/score/pull/64#discussion_r43220689>:

> @@ -21,6 +21,7 @@

>              <xsd:attribute type="xsd:boolean" name="externalDatabase" />

>              <xsd:attribute type="xsd:boolean" name="ignoreEngineJobs" />

>              <xsd:attribute type="xsd:boolean" name="registerPauseResumeService"/>

> +            <xsd:attribute type="xsd:boolean" name="registerWorkerNodeService"/>

Where do you use this flag and what for?

—
Reply to this email directly or view it on GitHub<https://github.com/CloudSlang/score/pull/64/files#r43220689>.